docs/README.md

```diff
@@ -1026,7 +1026,7 @@ export class MoveStock {
 }
 ```
 
-In this case, the client who submitted the command can still complete the operation. Then an event handler will take care of that `ErrorEvent` and proceed accordingly.
+In this case, the client who submitted the command can still complete the operation. Then, an event handler will take care of that `ErrorEvent` and proceed accordingly.
```

Comment:
`In this case, the command operation can still be completed. An event handler will take care of that `ErrorEvent` and proceed accordingly.`

Reply:
I like it. It is simpler and cleaner. Thank you :)